eh_win32_mingw.m

```diff
@@ -0,0 +1,56 @@
+#include "objc/runtime.h"
+#include "objc/objc-exception.h"
+#include "objc/hooks.h"
+#include <stdio.h>
+
+#include <windows.h>
+
+#define STATUS_GCC_THROW 0x20474343
+
+extern void *__cxa_current_exception_type(void);
+extern void __cxa_rethrow();
+
+BOOL handler_installed = NO;
+BOOL in_handler = NO;
```

Comment:
These probably need to be thread local.

Reply:
`in_handler`: yes, `handler_installed` tracks the call to `AddVectoredExceptionHandler` which should called only once per process?

Reply:
Makes sense, yes.